test/08_faucet_tests.ts

```diff
@@ -23,20 +24,21 @@ describe('Testing the Holograph Faucet', async () => {
   let dripCount = 0
 
   // Revert messages
-  const INITIALIZED = 'Faucet contract is already initialized'
+  const INITIALIZED = 'Faucet already initialized'
   const COME_BACK_LATER = 'Come back later'
   const NOT_AN_OWNER = 'Caller is not the owner'
 
   before(async function () {
     l1 = await setup()
-    SAMPLEERC20 = l1.sampleErc20.attach(l1.sampleErc20Holographer.address)
+    ERC20 = await l1.holographErc20.attach(l1.sampleErc20Holographer.address);
```

Comment:
`.attach` is not asynchronous

Reply:
Yep you're right, was doing copy paste and didn't pay it much attention. Will clean up the code from this in next iteration.